lib/clowder-common-ruby/types.rb

```diff
@@ -199,6 +199,7 @@ def valid_keys
         keys << :adminUsername
         keys << :adminPassword
         keys << :rdsCa
+        keys << :sslMode
```

Comment:
when I implemented Clowderization for Sources API few months ago, the sslMode was missing in some cdappconfig.json (I believe in ephemeral environment), just FYI, I haven't checked how it looks now

Reply:
@slemrmartin I can confirm as of today in ephem. clusters they do provide this, set to `disable`

```
oc -n ephemeral-29 rsh compliance-compliance-backend-57457cbfc5-b85n2
sh-4.4$ cat $ACG_CONFIG | jq '.' | grep -i sslmode
    "sslMode": "disable",
```